torch/jit/_serialization.py

```diff
@@ -182,3 +182,72 @@ def validate_map_location(map_location=None):
         validate_cuda_device(map_location)
 
     return map_location
+
+
+def jit_module_from_flatbuffer(f):
+    try:
+        import torch._C_flatbuffer as ff
+    except ImportError:
+        print("Please include //caffe2:_C_flatbuffer as dependency.")
```

Comment:
Is it possible for FlatBuffer dependency to be missing in OSS? If so, then we could probably have a dedicated error message for OSS